fauna/assets/logs/fauna.yaml

```diff
@@ -0,0 +1,104 @@
+id: fauna
+metric_id: fauna
+backend_only: false
+facets:
+  - description: Number of read operations consumed.
```

Comment:
For all these facets, there are specific to your integration. All facets defined here are using "global" path, meaning you are prefixing them. As they are specific to your integration, it would be great to prefix them by your integration name:
`path: BYTE_WRITE_OPS` -> `path: fauna.byte_write_ops`

You can find this information in our [public documentation](https://docs.datadoghq.com/developers/integrations/log_pipeline/#facet-requirements).

**Under Requirements:**
> All facets that are not mapped to reserved or standard attributes should be namespaced with the integration name.

Reply:
ok sounds good, I will add that, I didn't know if adding the group for it covered this.

Reply:
The group is having no impact on this. Thank you for making the update!

Reply:
@fauna-chase it looks like you actually updated the `name` to include `fauna` but we actually want to update `path`. Can you please update your facets accordingly?

Some info that might be helpful while doing so
* Name is just a display name that will show up in the app. This can even be something like `Fauna Byte Read Ops` if you want to make it user friendly. Path however refers to the path used when querying (eg searching for `@fauna.BYTE_WRITE_OPS:10`.
* To achieve this you will also need an `attribute-remapper` to remap `BYTE_WRITE_OPS` and other fields to be under `fauna`.
* You can see an example for this with the TypingDNA integration. The facet is defined [here](https://github.com/DataDog/integrations-extras/blob/c6bd9590cdb62e52dd32565999204d7c5f49592d/typingdna_activelock/assets/logs/typingdna.yaml#L10-L13) and it's remapped [here](https://github.com/DataDog/integrations-extras/blob/c6bd9590cdb62e52dd32565999204d7c5f49592d/typingdna_activelock/assets/logs/typingdna.yaml#L183-L192). While doing that you are also welcome to change the casing of this if you do not want the attributes to be uppercase.